src/constants/enums/fundingSources.ts

```diff
@@ -10,10 +10,7 @@ const fundingSources = [
   'Disc PI Medicare (MIP)',
   'Part D COB User Fees',
   'Exchange',
-  'Prog Ops',
```

Comment:
These deleted options are duplicates. I just happened to notice them. Maybe we need to confirm the options with stakeholders.